starlette_admin/base.py

```diff
@@ -133,7 +133,11 @@ def init_auth(self) -> None:
                 [] if self.middlewares is None else list(self.middlewares)
             )
             self.middlewares.append(
-                Middleware(AuthMiddleware, provider=self.auth_provider)
+                Middleware(
+                    AuthMiddleware,
+                    provider=self.auth_provider,
+                    allow_paths=[self.login_logo_url],
+                )
```

Comment:
```suggestion
                Middleware(AuthMiddleware, provider=self.auth_provider)
```
You can now allow additional paths from your AuthProvider by using the allow_paths parameter (#187). This makes it much easier. You can check out an example implementation at:  https://github.com/jowilf/starlette-admin/blob/6e07f49d11643a7bfca1fb391990500cc3ee095f/examples/auth/app.py#L46